tests/example_workflows/pyxaid2/example_1_Si/step4/run_step4.py

```diff
@@ -23,28 +23,28 @@
 
 params = {}
 
-params["nstates"] = 4
-params["nfiles"] = 30
+params["nfiles"]  = 1000  # Ex) # of Hvib files to read for a given traj
```

Comment:
again - either make this will break my examples. Perhaps, you could customize the scripts to be run depending on who is testing.